pkg/controller/nodes/dynamic/dynamic_workflow.go

```diff
@@ -116,6 +120,19 @@ func (d dynamicNodeTaskNodeHandler) buildDynamicWorkflowTemplate(ctx context.Con
 	}, nil
 }
 
+// Updates a dynamically generated subworkflow version to be deterministically generated from the computed closure.
+func setDeterministicVersion(ctx context.Context, workflowClosure *core.CompiledWorkflowClosure) error {
```

Comment:
this does not need to be done any more right? Why is this needed?

Reply:
reverted